exporter/exporterhelper/logs.go

```diff
@@ -43,6 +43,19 @@ func newLogsRequest(ctx context.Context, ld pdata.Logs, pusher consumerhelper.Co
 	}
 }
 
+func newLogsRequestUnmarshalerFunc(pusher consumerhelper.ConsumeLogsFunc) requestUnmarshaler {
+	return func(bytes []byte) (request, error) {
+		logs, err := pdata.LogsFromOtlpProtoBytes(bytes)
+		if err != nil {
+			return nil, err
+		}
+
+		// FIXME unmarshall context
```

Comment:
Why do we need to unmarshal the context?

Reply:
I think we don't use context in exporters currently and it's indeed not needed. I thought about being able to use this code in other processors and the case when e.g. `k8sprocessor` would be consuming the output. Then, this adds good amount of complexity. Also, e.g. `batchprocessor` does not retain context either. I think it should be removed altogether and mentioned as a limitation